src/custom/hooks/useFilterEmptyQueryParams.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo, useEffect } from 'react'
+import { useLocation, useHistory } from 'react-router-dom'
+
+function useQueryParams() {
```

Comment:
what about using the `useParsedQueryString` hook?

Reply:
I used.